test/unit/ux/table.test.ts

```diff
@@ -14,8 +13,7 @@ describe('table', function () {
       {baz: 7, foo: 'qux'},
     ]
     const columns = {baz: {header: 'Baz'}, foo: {header: 'Foo'}}
-    table(data, columns)
-
+    table(data, columns, {noStyle: true})
```

Comment:
shouldn't we have at least one test that includes passing in options? Also, shouldn't we have a test that checks that our default options are working properly?

Reply:
yeah, that's a good idea. i'll add a couple tests.

Reply:
ok, i've added a test for passing options. the default options for table is for styling-related, which means it'll have different behavior in a CI environment, so i didn't include a test for that.